plugins-scripts/Classes/F5/F5BIGIP/Component/LTM.pm

```diff
@@ -314,57 +323,45 @@ sub check {
   }
 }
 
-sub draw_html_table {
+sub build_table {
   my $self = shift;
+  my (@headers, @rows, $summary);
   if ($self->mode =~ /device::lb::pool::comple/) {
-    my @headers = qw(Node Port Enabled Avail Reason);
-    my @columns = qw(ltmPoolMemberNodeName ltmPoolMemberPort ltmPoolMbrStatusEnabledState ltmPoolMbrStatusAvailState ltmPoolMbrStatusDetailReason);
+    @headers = qw(Node Port Enabled Prio Rule Avail Reason);
+    my @columns = qw(ltmPoolMemberNodeName ltmPoolMemberPort ltmPoolMbrStatusEnabledState ltmPoolMemberPriority ltmPoolMemberMonitorRule ltmPoolMbrStatusAvailState ltmPoolMbrStatusDetailReason);
     if ($self->mode =~ /device::lb::pool::complections/) {
       push(@headers, "Connections");
       push(@headers, "ConnPct");
       push(@columns, "ltmPoolMemberStatServerCurConns");
       push(@columns, "ltmPoolMemberStatServerPctConns");
       foreach my $member (@{$self->{members}}) {
-        $member->{ltmPoolMemberStatServerPctConns} = sprintf "%.5f", $member->{ltmPoolMemberStatServerPctConns};
+        $member->{ltmPoolMemberStatServerPctConns} = sprintf "%.4f", $member->{ltmPoolMemberStatServerPctConns};
       }
     }
-    printf "<table style=\"border-collapse:collapse; border: 1px solid black;\">";
-    printf "<tr>";
-    foreach (@headers) {
-      printf "<th style=\"text-align: left; padding-left: 4px; padding-right: 6px;\">%s</th>", $_;
-    }
-    printf "</tr>";
+
+    $summary = sprintf "Pool mode: %s, Pool Monitor: %s", $_->{'ltmPoolLbMode'}, $_->{'ltmPoolMonitorRule'};
     foreach (sort {$a->{ltmPoolMemberNodeName} cmp $b->{ltmPoolMemberNodeName}} @{$self->{members}}) {
-      printf "<tr>";
-      printf "<tr style=\"border: 1px solid black;\">";
-      foreach my $attr (@columns) {
-        if ($_->{ltmPoolMbrStatusEnabledState} eq "enabled") {
-          if ($_->{ltmPoolMbrStatusAvailState} eq "green") {
-            printf "<td style=\"text-align: left; padding-left: 4px; padding-right: 6px; background-color: #33ff00;\">%s</td>", $_->{$attr};
-          } else {
-            printf "<td style=\"text-align: left; padding-left: 4px; padding-right: 6px; background-color: #f83838;\">%s</td>", $_->{$attr};
-          }
+      my %trh;
+      if ($_->{ltmPoolMbrStatusEnabledState} eq "enabled") {
+        if ($_->{ltmPoolMbrStatusAvailState} eq "green") {
+          $trh{style} = "border: 1px solid black; text-align: right; padding-left: 4px; padding-right: 6px; background-color: Lime;"
         } else {
-          printf "<td style=\"text-align: left; padding-left: 4px; padding-right: 6px; background-color: #acacac;\">%s</td>", $_->{$attr};
+          $trh{style} = "border: 1px solid black; text-align: right; padding-left: 4px; padding-right: 6px; background-color: Red;"
```

Comment:
This prepares data for the generic table_html function which now supports hash entry for the <tr> array.